cmd/edgectl/aes_login.go

```diff
@@ -32,17 +32,18 @@ func aesLogin(cmd *cobra.Command, args []string) error {
 	namespace, _ := cmd.Flags().GetString("namespace")
 	justShowURL, _ := cmd.Flags().GetBool("url")
 	showToken, _ := cmd.Flags().GetBool("token")
+	firstInstall, _ := cmd.Flags().GetBool("firstInstall")
```

Comment:
Is this left over testing code or is there a specific reason why we would want this flag in `edgectl login`?